dart/generate_map/lib/main.dart

```diff
@@ -0,0 +1,75 @@
+import 'dart:convert';
+import 'package:http/http.dart' as http;
+
+void returnFailure(final res, final msg) {
+  res.json({
+    "success": false,
+    "message": msg,
+  });
+}
+
+Future<void> start(final req, final res) async {
+  Map<String, dynamic> data = jsonDecode(req.payload);
```

Comment:
Would you please add error handling here in case the payload is not a valid JSON string?